packages/h/httpd/package.yml

```
@@ -20,7 +21,7 @@ builddeps  :
 setup      : |
     %apply_patches
     cat $pkgfiles/solus.layout >> config.layout
-    %configure \
+    ./configure $(sed 's|--runstatedir=/run||g' <<< "%CONFOPTS%") \
```

I *think* this should work as well:

```suggestion
    %reconfigure \
```

---

Yes, it works. But i got comment from Reily last time about changing recipe. He notes that i should add more comment. This is his suggestion about `%configure`, if that don't work that(`./configure bla bla`) is prefered because it shows clearly why it changed (runststedir error) rather than change it to `%reconfigure`. I don't mind changing it. Cheers!

---

Thanks for clarifying, that's a fair point. Leave it as is!